src/style.css

```diff
@@ -0,0 +1,57 @@
+/* layout */
+.benefits-section {
+    width: 87vw;
+    height: 655px;
+    background-color: var(--clr-ntrl-min); 
+}
+
+/*  section */
+.benefits-section {
+    position: relative;
+    top: -218px;
+    left: 7.4%;
+}
+
+.benefits-section .benefits-header {
+    color: #000;
+    padding-top: 93px;
+    padding-left: 99px;
+}
+
+.benefits-section .benefits-content {
+    display: flex;
+    width: 88%;
+    justify-content: space-between;
+    margin-top: 120px;   
+    margin-left: 100px;
+}
+
+/* components */
+.benefits-content .card-item {
+    position: relative;
+    width: 26%;
+    height: 280px;
+}
+
+
+.card-item  .card-header {
+    margin-top: 37px;
+}
+
+.card-item  .card-description {
+    margin-top: 26px;
+}
+
+.card-item  .card-detail {
+    display: flex;
+    position: absolute;
+    left: 0;
+    bottom: 0;
+    width: fit-content;
+    color: var(--clr-secondary-lt);
+    text-transform: uppercase;
+}
+
+.card-detail img[alt="arrow"] {
+    margin: auto 0px auto 26px;
+}
```

Comment:
checking editor config use 2 space instead 4 space, update for all

Reply:
thank you for your feedback and suggestions, I will modify it